project_tests.sh

```diff
@@ -4,4 +4,4 @@ ln -sf dashboard/dev_settings_test.py settings_test.py
 rm -rf build/junit.xml
 source venv/bin/activate
 python -m pytest --junitxml build/junit.xml dashboard/
-node_modules/grunt/bin/grunt test
+node_modules/.bin/grunt ci
```

Comment:
ok, so we need to install chromium on elife-dashboard `ci` and `end2end`

Reply:
Is that feasible? And can it be done in a fashion that it only effects those 2 environments and doesn't end up on prod?

Reply:
of course

Reply:
done, kicking build